src/main/webapp/WEB-INF/jsp/preview.jsp

```diff
@@ -246,7 +246,7 @@
                     <!-- Email preview table -->
                     <table id="email-list-table" class="list-group" role="list">
                         <tr rsf:id="row:" class="email-row list-group-item" role="listitem">
-                           <td class="right-content-email-toggle">
+                           <td class="right-content-email-toggle" role="presentation">
```

Comment:
@ChristianMurphy 
two things:
* We are in fluid in rsf way. so changing table tr to ul li is not the only things to do. if you do just that the email'list isn't rendered (I've tested that option).
* in fluid 1.1 (jQueryMobile view) there is [this](https://github.com/Jasig/email-preview/blob/506854d62aca7e1d43dc883869ebb8875ed2b515/src/main/webapp/js/email-browser.js#L592) in `email-browser.js`, And I wasn't sure of Fluid1.1's behavior. it seems to test and script `<table>` into a `<div>`. So with a `<table>` I'm confident in its behavior. and I need its CSS default display's behavior to let users zoom the email-list. On touch device it was working great whatever browser and OS. so -> keep `<table>` .
_(btw you can't zoom in email-list in google's Inbox native app, you can only zoom in the selected message...)_